src/components/InputField.tsx

```
@@ -33,23 +35,47 @@ const InputField : React.FC<InputProps> = ({
   customErrorMessage,
   containerStyle,
   errorStyle,
-  leftIcon
+  leftIcon,
+  resetKey
 }) => {
   const { theme, styles } = useCustomTheme(getInputStyles, 'Stock');
   const [errorMessage, setErrorMessage] = useState<string>('');
   const [isFocused, setIsFocused] = useState<boolean>(false);
```

ESLint

[error] 'isFocused' is assigned a value but never used. Allowed unused elements of array destructuring patterns must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] React Hook useEffect has a missing dependency: 'value'. Either include it or remove the dependency array. If 'setHasValue' needs the current value of 'value', you can also switch to useReducer instead of useState and read 'value' in the reducer.

(react-hooks/exhaustive-deps)